docs/how-to/how-to-use-virtio-fs-nydus-with-kata.md

```diff
@@ -0,0 +1,51 @@
+# Kata Containers with virtio-fs-nydus
+
+## Introduction
+
+Refer to [kata-nydus-design](../design/kata-nydus-design.md)
+
+## How to
+
+You can use Kata Containers with nydus as follows,
+
+1. Use [nydus latest branch](https://github.com/dragonflyoss/image-service);
+
+2. Deploy nydus environment as [Nydus Setup for Containerd Environment](https://github.com/dragonflyoss/image-service/blob/master/docs/containerd-env-setup.md);
+
+3. Start `nydus-snapshotter` with `enable_nydus_overlayfs` enabled;
+
+4. Use [kata-containers](https://github.com/kata-containers/kata-containers) `latest` branch to compile and build `kata-containers.img`;
+
+5. Update `shared_fs = virtio-fs-nydus`, `virtio_fs_daemon = <nydusd binary path>`, `virtio_fs_extra_args = []` in `configuration-qemu.toml`;
+
+6. run `crictl run -r kata-qemu nydus-sandbox.yaml nydus-container.yaml`;
+
+The `nydus-sandbox.yaml` looks like below:
+
+```yaml
+metadata:
+  attempt: 1
+  name: nydus-sandbox
+  namespace: default
+log_directory: /tmp
+linux:
+  security_context:
+    namespace_options:
```

Comment:
Is network namespace required to run this example? If not, it's better to delete it, or users may think this is a requirement.

Reply:
I think here network namespace is required and should set `host` network. Without this network namespace setting, containerd will use bridge network default and maybe users do not have bridge network environment.